tests/test_data.py

```diff
@@ -319,6 +330,60 @@ def create_MeanResponseTransformer_test_df_unseen_levels():
     return df
 
 
+def create_OneHotEncoderTransformer_test_df_1():
```

Comment:
S: If these are specific to the OneHotEncoderTransformer tests I think it might be better to create them in that test module.

Reply:
Hey David - there are many functions in here which seem to be specific to only one test module e.g. 'create_datediff_test_df', 'create_date_test_df', 'create_NearestMeanResponseImputer_test_df'. Would you prefer that I just move the OHE specific functions here and then create a fast follow ticket to move the others? Or I could do them all as part of this PR?

Reply:
Yes please do that.  I think that fixtures should probably be co-located with tests in the scope they serve.

Reply:
Dealt with that in [this](https://github.com/lvgig/tubular/pull/267/commits/b6488c504bf8a80e55900478ac285545b9c0d7bd) commit. If a fixture is used in just a single test have moved it to that test.